doc/getting_started/installation_win.rst

```diff
@@ -88,6 +88,11 @@ packages from their respective websites.
       cd %userprofile%\zephyr
       pip install --user -r scripts/requirements.txt
 
+#. If you need to install pyocd, it is available as a python 2 module only, this
```

Comment:
And here:

    #. If you require pyocd, an open source python2 library for programming and
       debugging ARM Cortex-M microcontrollers, use this command::